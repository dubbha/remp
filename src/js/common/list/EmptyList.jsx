import React from 'react';

const EmptyList = () => (
  <div className="list__message">
    No films found
  </div>
);

export default EmptyList;
