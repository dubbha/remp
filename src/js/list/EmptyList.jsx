import React from 'react';

const List = () => (
  <div className="list__message">
    No films found
  </div>
);

export default List;
