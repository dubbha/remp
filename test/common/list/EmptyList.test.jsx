import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import EmptyList from 'common/list/EmptyList';

describe('EmptyList', () => {
  it('should render successfully', () => {
    const wrapper = shallow(<EmptyList />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
