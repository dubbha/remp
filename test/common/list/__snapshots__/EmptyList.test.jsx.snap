// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyList should render successfully 1`] = `
<div
  className="list__message"
>
  No films found
</div>
`;
